Formula/qcachegrind.rb

```diff
@@ -12,16 +11,16 @@ class Qcachegrind < Formula
     sha256 "d16d0cd82b7f7e69862a636c70fb72aaddb450fdaea8653a77486e321ef811b6" => :yosemite
   end
 
-  depends_on "graphviz" => :optional
   depends_on "qt"
+  depends_on "graphviz" => :optional
 
   def install
-    cd "qcachegrind"
-    system "#{Formula["qt"].bin}/qmake", "-spec", "macx-clang", "-config", "release"
-    system "make"
-    # Install app
-    prefix.install "qcachegrind.app"
-    # Symlink in the command-line version
-    bin.install_symlink prefix/"qcachegrind.app/Contents/MacOS/qcachegrind"
+    cd "qcachegrind" do
```

Comment:
The only thing I find particularly gross is if we end up doing things like this:

```ruby
if build.head?
  cd "horse" do
    system "autoreconf", "-fvi"
    system "./configure", "--blah"
    system "make", "install"
else
  system "./configure", "--blah"
  system "make", "install"
end
```

That's why I always let `cd "horse" if build.head?` kind of things go, but beside that I've no strong preference either side of the line. Just let me know what you like.